download/model/schedule.py

```diff
@@ -0,0 +1,131 @@
+    'Entity_Cd': 'string',
+    'Tran_NamL': 'string',
+    'Tran_NamF': 'string',
+    'Tran_NamT': 'string',
+    'Tran_NamS': 'string',
+    'Tran_Adr1': 'string',
+    'Tran_Adr2': 'string',
+    'Tran_City': 'string',
+    'Tran_State': 'string',
+    'Tran_Zip4': 'string',
+    'Tran_Emp': 'string',
+    'Tran_Occ': 'string',
+    'Tran_Self': bool,
+    'Tran_Type': 'string',
+    'Tran_Date': 'string',
+    'Tran_Date1': 'string',
+    'Tran_Amt1': float,
+    'Tran_Amt2': float,
+    'Tran_Dscr': 'string',
+    'Cmte_ID': 'string',
+    'Tres_NamL': 'string',
+    'Tres_NamF': 'string',
+    'Tres_NamT': 'string',
+    'Tres_NamS': 'string',
+    'Tres_Adr1': 'string',
+    'Tres_Adr2': 'string',
+    'Tres_City': 'string',
+    'Tres_State': 'string',
+    'Tres_Zip': 'string',
+    'Intr_NamL': 'string',
+    'Intr_NamF': 'string',
+    'Intr_NamT': 'string',
+    'Intr_NamS': 'string',
+    'Intr_Adr1': 'string',
+    'Intr_Adr2': 'string',
+    'Intr_City': 'string',
+    'Intr_State': 'string',
+    'Intr_Zip4': 'string',
+    'Intr_Emp': 'string',
+    'Intr_Occ': 'string',
+    'Intr_Self': bool,
+    'Cand_NamL': 'string',
+    'Cand_NamF': 'string',
+    'Cand_NamT': 'string',
+    'Cand_NamS': 'string',
+    'tblDetlTran_Office_Cd': 'string',
+    'tblDetlTran_Offic_Dscr': 'string',
+    'Juris_Cd': 'string',
+    'Juris_Dscr': 'string',
+    'Dist_No': 'string',
+    'Off_S_H_Cd': 'string',
+    'Bal_Name': 'string',
+    'Bal_Num': 'string',
+    'Bal_Juris': 'string',
+    'Sup_Opp_Cd': 'string',
+    'Memo_Code': 'string',
+    'Memo_RefNo': 'string',
+    'BakRef_TID': 'string',
+    'XRef_SchNm': 'string',
+    'XRef_Match': 'string',
+    'Loan_Rate': 'string',
+    'Int_CmteId': 'Int64'
+}
+
+class ScheduleBase:
+    '''
+    Sets common schema for Form 460 Schedule data
+    '''
+    def __init__(
+        self,
+        form_id: str,
+        transactions: pl.DataFrame,
+        filings: pl.DataFrame,
+        committees: pl.DataFrame
+    ):
+        schedule = committees.lazy().group_by('Filer_ID').first().join(
+            filings.lazy(),
+            on='filer_nid',
+            how='inner'
+        ).rename({
+            '_Committee_Type': 'Committee_Type'
+        }).join(
+            transactions.lazy().filter(pl.col('cal_tran_type') == form_id),
+            on='filing_nid',
+            how='inner'
+        ).drop([
+            'filing_nid', # TODO: Figure out which df's these cols come from and drop them before join
+            'filer_nid',
+            'data_warning',
+            'candidate_controlled_id',
+            'cal_tran_type',
+            '_Status',
+            'Support_Or_Oppose',
+            'Start_Date',
+            'Make_Active',
+            'End_Date',
+            'Ballot_Measure',
+            'Ballot_Measure_Election'
+        ])
+
+        self._lazy = schedule
+
+        self._dtypes = DTYPES
+
+    @property
+    def lazy(self):
+        return self._lazy
```

Comment:
Do we need to expose this?  Or will you mainly be wanting to call the `df` method?  It might be good to make use of the objects to hide the `lazy` logic unless you have a need to expose it to the outside.

Reply:
I think it's a good idea to expose lazy because lazy is preferred for operating on, especially if there's an opportunity for chaining